specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -4265,7 +4339,11 @@
         },
         "vmCustomConfigurationUri": {
           "type": "string",
-          "description": "The uri to the storage blob containing scripts to be run on the virtual machine after provisioning."
+          "description": "The uri to the storage blob containing the arm template to be run on the virtual machine after provisioning."
+        },
+        "vmCustomConfigurationParameterUri": {
```

Comment:
Sorry, I have a comment for MarketPlaceInfoProperties.properties.exactVersion.  This field is required, but currently we don't have exactVersion in existing VMTemplate.  How do the data migration works?  Please clarify.  thanks.

Reply:
can you have the ui ask user to pick an exact version?

Reply:
need to talk to ryan too.

Reply:
ok I figured out the version format from https://docs.microsoft.com/en-us/azure/virtual-machines/image-version-vm-cli#create-the-image-version. I should be able to get latest version with given publisher name, offer and sku.

Reply:
Please close this comment